test/test_helper.rb

```diff
@@ -4,27 +4,22 @@
 require 'minitest/autorun'
 require 'toxiproxy'
 
-TINYTDS_SCHEMAS = ['sqlserver_2000', 'sqlserver_2005', 'sqlserver_2008', 'sqlserver_2014', 'sqlserver_azure', 'sybase_ase'].freeze
+TINYTDS_SCHEMAS = ['sqlserver_2016', 'sqlserver_azure'].freeze
```

Comment:
Similar question about saying 2016 in this file if the new min version is 2017

Reply:
addressed with 6684afba.